patch/kconfig-inclusions

```diff
@@ -126,6 +126,182 @@ CONFIG_SENSORS_EMC2305=m
 [marvell-arm64]
 CONFIG_MARVELL_AC5_WATCHDOG=m
 
+[pensando]
```

Comment:
i do not quite understand this section now. when do we use this section in the make process? for example, when we build marvell-arm64, will it have pensando config section.

in the makefile, how do we specifiy which section to use? 

@

Reply:
@saiarcot895 , do you know?

Reply:
The `common`,  `$ARCH` (which should be `arm64` in this case), and `$PLATFORM` (which should be `pensando` in this case) sections would get included through `manage-config` . `$ARCH` and `$PLATFORM` come from sonic-buildimage.

Unless there are changes on that side, these configs won't get included.

Reply:
@saiarcot895 @lguohan 
specifying [PLATFORM-ARCH] way in kconfig-inclusion/kconfig-exclusion is broken. I see that it works if we specify the platform (without platform-arch).. 
This is due to https://github.com/sonic-net/sonic-linux-kernel/blob/e7c93a542272b5952a1082326c097ec7633f881b/manage-config#L64

Other platforms (for ex: marvell-arm64) might need to fix it.

Reply:
ok.. i see platform name itself is marvell-arm64 in https://github.com/pensando/sonic-buildimage/tree/master/platform .. So it works fine.. i thought the convention is "platform-arch"